odd-platform-api/src/main/java/com/provectus/oddplatform/mapper/TagMapperImpl.java

```diff
@@ -52,4 +53,13 @@ public TagsResponse mapTags(final List<Tag> tags) {
             .items(tags)
             .pageInfo(pageInfo(tags.size()));
     }
+
+    public Tag mapTag(final TagDto dto) {
+        final Tag tag = new Tag();
```

Comment:
Can be chained:
```
new TagPojo()
            .setName(form.getName())
            .setImportant(form.getImportant());
```

Reply:
`TagPojo` can be, but `Tag`'s setters return void.

Reply:
my bad, that'd be
```
        return new Tag()
            .id(pojo.getId())
            .name(pojo.getName())
            .important(pojo.getImportant());
```

Reply:
Done